From jwomack@eecs.wsu.edu  Tue Mar 30 16:05:04 1999
Return-Path: <jwomack@eecs.wsu.edu>
Received: from mail.redhat.com (mail.redhat.com [199.183.24.239])
	by developer.redhat.com (8.8.7/8.8.7) with ESMTP id QAA02309
	for <bugzilla@developer.redhat.com>; Tue, 30 Mar 1999 16:05:04 -0500
Received: from lacrosse.redhat.com (root@lacrosse.redhat.com [207.175.42.154])
	by mail.redhat.com (8.8.7/8.8.7) with ESMTP id QAA02564
	for <bugzilla@developer.redhat.com>; Tue, 30 Mar 1999 16:05:03 -0500
Received: from mail.redhat.com (mail.redhat.com [199.183.24.239])
	by lacrosse.redhat.com (8.8.7/8.8.7) with ESMTP id QAA09607
	for <bugzilla@lacrosse.redhat.com>; Tue, 30 Mar 1999 16:05:01 -0500
Received: from dns1.eecs.wsu.edu (root@dns1.eecs.wsu.edu [134.121.64.1])
	by mail.redhat.com (8.8.7/8.8.7) with ESMTP id QAA02552
	for <bugzilla@redhat.com>; Tue, 30 Mar 1999 16:04:58 -0500
Received: from eecs.wsu.edu (IDENT:jwomack@roberts.eecs.wsu.edu [134.121.66.23]) by dns1.eecs.wsu.edu with ESMTP (8.9.3/)
	id NAA18173 for <bugzilla@redhat.com>; Tue, 30 Mar 1999 13:04:55 -0800 (PST)
Sender: jwomack@eecs.wsu.edu
Message-ID: <37013C76.518F0EC1@eecs.wsu.edu>
Date: Tue, 30 Mar 1999 13:04:54 -0800
From: Joseph Womack <jwomack@eecs.wsu.edu>
Organization: Washington State University
X-Mailer: Mozilla 4.08 [en] (X11; I; Linux 2.2.1 i586)
MIME-Version: 1.0
To: bugzilla@redhat.com
Subject: BUG ID #1894
Content-Type: multipart/mixed; boundary="------------957C7BD46B93451A1A635699"

This is a multi-part message in MIME format.
--------------957C7BD46B93451A1A635699
Content-Type: text/plain; charset=us-ascii
Content-Transfer-Encoding: 7bit

This should fix the problem.  It seems to for me :-)

Joe

----------------------------------------------------------------------------
Joe Womack					Washington State University
jwomack@eecs.wsu.edu				EECS System Staff
--------------957C7BD46B93451A1A635699
Content-Type: text/plain; charset=us-ascii; name="gated-3-5-10-linux-v2.2.patch"
Content-Transfer-Encoding: 7bit
Content-Disposition: inline; filename="gated-3-5-10-linux-v2.2.patch"

--- gated-3-5-10/src/krt_rtread_proc.c.orig	Tue Mar 30 09:17:49 1999
+++ gated-3-5-10/src/krt_rtread_proc.c	Tue Mar 30 12:56:13 1999
@@ -125,9 +125,18 @@
  *     each line. This changed  reader will always stay in sync.
  *             -- Alan Cox.
  */
+
+/*
+ *     Despite Alan's comment, they did indeed change the format of /proc
+ *     /proc/net/route.  They changed the flags to be 4 digits instead of
+ *     the 2 digits
+ *
+ *            -- Joe Womack <jwomack@eecs.wsu.edu>
+ */
+
     while(fgets(buf,256,DFile))
     {
-       i=sscanf(buf,"%s\t%lx\t%lx\t%02x\t%*d\t%*d\t%*d\t%lx\t%*d\t%*d\n",
+       i=sscanf(buf,"%s\t%lx\t%lx\t%04x\t%*d\t%*d\t%*d\t%lx\t%*d\t%*d\t%*d\n",
 		 if_name,
 		 &addr_dest,
 		 &addr_gw,

--------------957C7BD46B93451A1A635699--

